mysql/time.go

```diff
@@ -1375,3 +1375,40 @@ func ExtractTimeValue(unit string, format string) (int64, int64, int64, time.Dur
 		return 0, 0, 0, 0, errors.Errorf("invalid singel timeunit - %s", unit)
 	}
 }
+
+// IsClockUnit returns true when unit is interval unit with hour, minute or second.
+func IsClockUnit(unit string) bool {
+	switch strings.ToUpper(unit) {
+	case "MICROSECOND", "SECOND", "MINUTE", "HOUR":
+		fallthrough
```

Comment:
`fallthrough` is unnecessary, we can just list the rest of the cases in a new line.